aiida/scheduler/__init__.py

```diff
@@ -145,11 +145,10 @@ def get_submit_script(self, job_tmpl):
 
         empty_line = ""
 
-        shebang = "#!/bin/bash"
 
         # I fill the list with the lines, and finally join them and return
         script_lines = []
-        script_lines.append(shebang)
+        script_lines.append(job_tmpl.shebang)
```

Comment:
Can we put here a default? Something like: `if job_tmpl.shebang is None: ... else: ...`? I see you changed all plugins, which is good, but I think there are a few that are developed as external plugins and might not be aware of this change.

Reply:
(maybe verifying that the default is None, and leaving the option for an empty string to be dealt differently, if a user does not want the shebang line at all? I don't know if there is ever this need, but one never knows). Something like `elif job_tmpl == "": #dont print anything, not even an empty line`